aws_tools/dynamodb_handler.py

```diff
@@ -77,48 +77,53 @@ def delete_item(self, keys):
         )
 
     def query_items(self, query=None, only_fields_with_values=True):
-        filter_expression = ''
+        filter_expression = None
         if query and len(query) > 1:
             for field, value in iteritems(query):
-                value = None
```

Comment:
If we immediately reassign `value` to `None`, we discard the actual contents of the `value` variable. Before removing this line, `isinstance(value, dict)` would always evaluate to `False`.